### PR TITLE
1.14/laravel 9

### PR DESCRIPTION
**Pull request by neonleann**
https://trello.com/c/JxAnoTB3/823-l9-minimal-upgrade

Most of this is pretty straight forward. You may want to look at [Fix testItCanDecryptAStoredZip.](https://github.com/neontribe/ARCVService/commit/f19efa66c3648464949f423070a1db4a49c7fd02). It all seems to work, but a second look would be good.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,47 +12,45 @@
...
     "laravel/tinker": "^2.7.2",
+    "laravel/ui": "^4.1",
     "maennchen/zipstream-php": "^2.1.0",
-    "moontoast/math": "^1.2.1",
     "neontribe/laravel-specification": "dev-master",
-    "ramsey/uuid": "^3.9.6",
-    "sebdesign/laravel-state-machine": "^2.1.1",
-    "symfony/event-dispatcher": "^4.4.37",
-    "symfony/console": "4.4.36",
+    "ramsey/uuid": "4.7.1",
```
Comment (charlesstrange2) [inline composer.json:33]: check this is actually needed as a top level dependency.
I think laravel may pull it in now.

File: composer.json [hunk 2]
```diff
@@ -12,47 +12,45 @@
...
+    "sebdesign/laravel-state-machine": "3.3.0",
```
Comment (charlesstrange2) [inline composer.json:34]: check changelog on this; it's quite important.

**Review by charlesstrange2 — APPROVED** [review]
This looks good, just a couple of dependencies to check the updates on.

**Comment by lauragurney** [comment]
Superseded by https://github.com/neontribe/ARCVService/pull/592. Closing this PR as the newer one has been merged into 1.15/release.

**Pull request closed without merging**